lib/tasks/fetch.rake

```diff
@@ -128,3 +128,7 @@ end
 task log_daily_usage_measurables: :environment do
   Metrics::RecordDailyUsageWorker.perform_async
 end
+
+task log_daily_database_and_elasticsearch_record_counts: :environment do
+  Metrics::RecordDbAndEsRecordCountsWorker.perform_async
```

Comment:
I could see a case where this namespace isn't the best. While the other workers in `Metrics` actually log using Timber as the task names suggest, the behavior is still very similar and eventually, we'll be sending all logs to Datadog anyways.

Reply:
If you pick up my name suggestion above, maybe `Search::ReconciliationWorker` or `Search::Statsworker` could be alternatives?